bazel/repos.bzl

```diff
@@ -37,12 +36,11 @@ def repos(external = True, repo_mapping = {}):
             sha256 = "13e7c6460cd979726e5b3b129bb01c34532f115883ac696a75eb7f1d6a9765ed",
         )
 
-
     if external and "com_github_3rdparty_stout_grpc" not in native.existing_rules():
         git_repository(
             name = "com_github_3rdparty_stout_grpc",
             remote = "https://github.com/3rdparty/stout-grpc",
             commit = "fef528dea479b09dec7bc49aa7535c0e87bca310",
-            shallow_since = "1632152723 -0700"
+            shallow_since = "1632152723 -0700",
```

Comment:
This is the failing one I assume. My bad, could have sworn that I had tested it. Sorry about that.